srcpkgs/luksmeta-devel

```diff
@@ -0,0 +1 @@
+luksmeta/
```

Comment:
symlink has a slash in it, might want to remove the /

Reply:
Done.